.circleci/apps.yml

```diff
@@ -30,6 +30,7 @@ jobs:
   check-changelog:
     # We use the machine executor, i.e. a VM, not a container
     machine:
+      image: ubuntu-2004:202201-02
```

Comment:
IMO you don't need a machine executor to check the changelog. In most of our projects we are using the docker image `circleci/buildpack-deps:stretch-scm`

Reply:
Fixed!